### PR TITLE
Docs/readme

### PR DESCRIPTION
**Pull request by daltobello**
Updated our README to include preview images of our app!

### CODE REVIEW

**Comment by jenziel** [comment]
hey devin before I update this can you add line breaks between our github usernames?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- Have fun recording your ideas!
+### Preview of App:
+![App preview image 1](assets/App_Preview_01.png)
+![App preview image 2](assets/App_Preview_02.png)
+
+### Context:
+- We spent 20-25 hours working on this project. We are in week four of Mod 1.
+
+### Contributors:
+https://github.com/LDeLaRosa13
```
Comment (jenziel) [inline README.md:21]: can you add line breaks in between each of these? 

> To create a line break or new line, end a line with two or more spaces, and then type return.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- The ability to favorite idea cards when a user clicks the star button
+- The ability to filter only the favorited ideas
+
+### Installation Instructions:
+- Git clone this repo to your local machine
+- Cd into that directory
+- Enter the command open index.html to open the browser
+- Have fun recording your ideas!
+### Preview of App:
+![App preview image 1](assets/App_Preview_01.png)
```
Comment (jenziel) [inline README.md]: I like that you used the assets file path to insert these in.  When i used git fetch to look at this, those png files weren't showing up in the assets folder though.

File: README.md (hunk 1 above)
Comment (daltobello) [reply README.md:21]: Yes! Good call. I just added line breaks and pushed this branch again.

**Pull request merged**